package.json

```diff
@@ -112,7 +112,6 @@
     "code-lint": "eslint setup lib bin hot buildin \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\"",
     "type-lint": "tsc --pretty",
     "fix": "yarn code-lint --fix",
-    "pretty": "prettier \"setup/**/*.js\" \"lib/**/*.js\" \"bin/*.js\" \"hot/*.js\" \"buildin/*.js\" \"test/*.js\" \"test/**/webpack.config.js\" \"examples/**/webpack.config.js\" \"schemas/**/*.js\" \"declarations.d.ts\" --write",
```

Comment:
You Want to keep the npm script

Reply:
It is not redundant, one is ESLint other is Prettier

Reply:
As per @sokra explanation in #7276 we're using prettier plugin for ESLint, meaning ESLint runs prettier as a rule and reports errors/warnings. Also, ESLint will format code based on prettier rules when `--fix` flag is used, meaning does exactly the same as `pretty` script does.

Taking into consideration the above and the fact that `pretty` script remains unused I thought that we could get rid of it unless someone is using `yarn pretty` explicitly, but I doubt that.

Reply:
I use `yarn pretty` explicitly. It's faster than running eslint and doesn't do non-prettier changes.